src/packages.builds

```diff
@@ -8,14 +8,31 @@
     <BuildAllOSGroups Condition="'$(FilterToOSGroup)' != ''">false</BuildAllOSGroups>
   </PropertyGroup>
 
+  <!--
+    the following packages are built and shipped from TFS, so while we still want
+    to build them we don't want to ship them.  by setting NonShippingPackage to true
+    the packages will still be built but placed in a different location.
+  -->
   <ItemGroup>
-    <Project Include="Native\pkg\**\*.builds" Condition="'$(SkipNativePackageBuild)' != 'true'">
+    <NativeNonShippingPackage Include="Native\pkg\runtime.native.System.IO.Compression\*.builds" />
+    <NonShippingPackage Include="System.IO.Compression\pkg\*.builds" />
```

Comment:
I would expect that only the native packages should need to be skipped. Why are we shipping the managed package as well?